<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtendedSessionListView</class>
 <widget class="QWidget" name="ExtendedSessionListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExtendedSessionListView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
    font-family:'微软雅黑';
}

QPushButton#createnewgroup {
    border: 1px solid #00000000;
    image: url(:/Images/addButton.png);
}


QLabel#titleText
{
	border:none;
	color:#CFCFCF;
    font-size:14px;
}

QPushButton#pb_back{ 
    border:none;
	image : url(:/Images/previous.png);
} 
QPushButton#pb_back:hover{ 
    image : url(:/Images/previous_hover.png);
} 
QPushButton#pb_back:pressed{ 
    image : url(:/Images/previous_hover.png);
} 
                                    </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="title_panel" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pb_back">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleText">
        <property name="text">
         <string>订阅号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_left_2">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="localman.qrc"/>
 </resources>
 <connections/>
</ui>
